src/components/AssetSearch/stories/AssetSearch.stories.tsx

```diff
@@ -11,53 +18,71 @@ import customStyles from './customStyles.md';
 import empty from './empty.md';
 import error from './error.md';
 import full from './full.md';
-import rootAssetSelect from './rootAssetSelect.md';
+import handleSearchResults from './handleSearchResults.md';
 
 // Mock the SDK calls
-sdk.Assets.list = async (
-  input: sdk.AssetListParams
-): Promise<sdk.AssetDataWithCursor> => {
-  action('sdk.Assets.list')(input);
-  return {
-    items: assetsList.map(
-      (a: sdk.Asset): sdk.Asset => {
-        return {
-          id: a.id,
-          name: a.name,
-          description: a.description,
-        };
+export const fakeClient: API = {
+  // @ts-ignore
+  assets: {
+    // @ts-ignore
+    list: (scope: AssetListScope) => {
+      action('assets.list')(scope);
+      // @ts-ignore
+      // pick only required fields
+      const items: Asset[] = assetsList.map(asset =>
+        pick(asset, [
+          'id',
+          'name',
+          'description',
+          'lastUpdatedTime',
+          'createdTime',
+          'depth',
+          'paths',
+        ])
+      );
+      return new Promise(resolve => {
```

Comment:
`assets.list` should return `CogniteAsyncIterator<Asset>` that has method `autoPagingToArray`

Reply:
It (`asset.list`) actually uses by RootAssetSelect. Since you've disabled it, you shouldn't use this mock at all

Reply:
well, it's supposed to be here when we restore RootAssetSelect. I could comment it out for a while as all other changes for RootAssetSelect